app/Rules/Booking/UserHasNotActiveBookingsForTrip.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Rules\Booking;
+
+use App\User;
+use App\Models\Trip;
+use App\Models\Booking;
+use App\Services\Requests\CreateBookingRequest;
+
+class UserHasNotActiveBookingsForTrip implements CreateBookingRule
+{
+    public function validate(Trip $trip, User $user, CreateBookingRequest $request): bool
+    {
+        if ($trip->bookings()->where(['user_id' => $user->id])->where('status', '!=', Booking::STATUS_DECLINED)->count()) {
```

Comment:
its better to create repository method in order to calculate bookings count related to user